File: golang/GoCode/src/gk001/ch1/helloworld.go
package main

import (
	"fmt"
	"os"
)

func main() {
	var a int = 2
	var b int = 3
	c := a + b
	fmt.Println("hello,世界 ", c)
	fmt.Println(os.Args)
	if len(os.Args) > 1 {
		fmt.Println("hello world", os.Args[1])
	}
	fmt.Print("hello world!\n")
	fmt.Println("hello world012!")
	os.Exit(255)

}

File: last_tobeEngineer/javacode/src/helloworld.java
import java.util.logging.Logger;

public class helloworld {

    private static final String TAG = "helloworld";
    public static Logger log = Logger.getLogger(helloworld.class.toString());
    public static void main(String[] args) {
//        System.out.println("hello world");
//        System.out.println(args[0]);
//        System.out.println(args[1]);
        father fatherA = new father();
        fatherA.name = "zhang 3";
        fatherA.age = 26;
        fatherA.child = new ChildClass();
        fatherA.child.name = "xiao zhang 3";
        fatherA.child.age = 5;

        father fatherB = (father) fatherA.clone();
        log.info("fatherA=fatherB is "+(fatherA==fatherB));
        log.info("fatherA.hashCode="+fatherA.hashCode());
        log.info("fine:"+fatherA.name);
        log.info("fine:"+fatherA.age);
        log.info("fatherB.hashCode="+fatherB.hashCode());
        log.info("fine:"+fatherB.name);
        log.info("fine:"+fatherB.age);


    }


}




